WebGoatCore/Views/Checkout/Receipt.cshtml

```diff
@@ -10,51 +10,51 @@
         @if (Model != null)
         {
             <p>Order: @Model.OrderId</p>
-            <table class="Receipt">
+            <table class="receipt">
                 <tbody>
-                    <tr>
-                        <th>Product Id</th>
-                        <th>Product Name</th>
-                        <th>Unit Price</th>
-                        <th>Quantity</th>
-                        <th>Extended Price</th>
+                    <tr class="headerRow">
+                        <th class="tabHeader">Product Id</th>
+                        <th class="tabHeader">Product Name</th>
+                        <th class="tabHeader">Unit Price</th>
+                        <th class="tabHeader">Quantity</th>
+                        <th class="tabHeader">Extended Price</th>
```

Comment:
I think there is no need to add class to this th elements. The headerRow already defines the row as the header, so you should be able to apply CSS with something like `tr.headerRow th` (https://www.w3schools.com/css/css_combinators.asp)